service_capacity_modeling/hardware/profiles/pricing/aws/3yr-reserved_zz-overrides.json

```diff
@@ -0,0 +1,68 @@
+{
+  "us-east-1": {
+    "instances": {
+      "f5.large.deprecated": {"annual_cost": 300.0, "lifecycle": "deprecated"},
+      "m6id.xlarge":   {"annual_cost": 781.66},
+      "m6id.4xlarge":  {"annual_cost": 3126.64},
```

Comment:
Just to call out, these appear to me to differ from the API fetch. Changing these to the current dynamically loaded values causes a failure on `tests/netflix/test_cassandra.py::test_capacity_high_writes`. I believe this is because it shifts the relative pricing, which shifts number of expected instances in the assertion.

@jolynch recommend if you can validate that, and we can fast-follow to remove these overrides.

Reply:
Yup I can own that, I'm 99% sure it's fine.

Reply:
I tidied up and added explicit tests on the sort. I had to comment out this test, to be fixed. It turns out the failing test is because of the sort order in the input!, I wasn't able to do both sort and have this test pass. In other words, I no longer think this is because of the relative pricing shift, it seems to be more the traversal order of instance type names.